src/app/core/entity-components/entity-details/form/form.stories.ts

```diff
@@ -0,0 +1,97 @@
+import { FormComponent } from "./form.component";
+import { moduleMetadata } from "@storybook/angular";
+import { Meta, Story } from "@storybook/angular/types-6-0";
+import { EntityMapperService } from "../../../entity/entity-mapper.service";
+import { AlertService } from "../../../alerts/alert.service";
+import { ChildPhotoService } from "../../../../child-dev-project/children/child-photo-service/child-photo.service";
+import { SessionService } from "../../../session/session-service/session.service";
+import { EntityDetailsModule } from "../entity-details.module";
+import { Child } from "../../../../child-dev-project/children/model/child";
+import { RouterTestingModule } from "@angular/router/testing";
+import { User } from "../../../user/user";
+import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
+import { EntityPermissionsService } from "../../../permissions/entity-permissions.service";
+
+export default {
```

Comment:
There is a problem when I run this in storybook: sometimes I get an error that the `EntityDetailsComponent` is part of two modules. Any idea why this happens?

Reply:
Do you get this after the storybook is reloaded live when changing the code? I have the same problem (I think some bug in storybook itself) but reloading with F5 works

Reply:
Yes thats exactly the behavior, so then I guess it is alright.